logger.go

```diff
@@ -11,6 +12,7 @@ import (
 	"github.com/go-stack/stack"
 
 	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger/loggercontext"
```

Comment:
I don't like this isn't in a top level package. Why to force users to make another import?

Reply:
Number of imports is not an argument to me. Your editor can sort it automatically.

Reply:
Your editor won't tell you about the api. You have to know that before it can sort imports for you. For me making several packages in a repo that does so simple thing smells a bad design. Imagine how many packages there would be in Go stdlib.